File: 프로그래머스/LV0/코딩테스트 입문/순서쌍의 개수/순서쌍의 개수.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <math.h>
int solution(int n) 
{
    int answer = 0;
    double root_n = pow(n, 0.5);
    for(int i = 1; i <= (int)root_n; i++)
    {
        if(n % i == 0)answer += 1;
    }
    return root_n == (int)root_n ? (answer-1) * 2 + 1 : answer * 2;
}

File: 프로그래머스/LV0/코딩테스트 입문/순서쌍의 개수/순서쌍의 개수.py
def solution(n):
    answer = len(list(filter(lambda x : n % x == 0, range(1, int(n**(0.5))+1))))
    return (answer-1) * 2 + 1 if n**(0.5) == int(n**(0.5)) else answer * 2

# def solution(n):
#     answer = 0
#     for i in range(1, int(n**(0.5))+1):
#         if n % i == 0:
#             answer += 1
#     return (answer-1) * 2 + 1 if n**(0.5) == int(n**(0.5)) else answer * 2